src/registry.jl

```diff
@@ -26,6 +27,8 @@ add_format(format"JLD2", (unsafe_wrap(Vector{UInt8},"Julia data file (HDF5), ver
 add_format(format"GZIP", [0x1f, 0x8b], ".gz", [:Libz => UUID("2ec943e9-cfe8-584d-b93d-64dcb6d567b7")])
 add_format(format"BSON",(),".bson", [:BSON => UUID("fbb218c0-5317-5bc6-957e-2ee96dd4b1f0")])
 add_format(format"JLSO", (), ".jlso", [:JLSO => UUID("9da8a3cd-07a3-59c0-a743-3fdc52c30d11")])
+add_format(format"NPY", "\x93NUMPY", ".npy", [idNPZ])
+add_format(format"NPZ", "", ".npz", [idNPZ])
```

Comment:
Is it always a gzip file here? If so might be worth using the gzip magic bytes.

Reply:
I just checked:

```python
np.savez("test.npz", a=np.array([1,2,3]), b=np.array([1.2, 3.4]))
```

Generated file looks just like a plain `.zip` file for me. 
The file magic is just zip file magic "PK\x03\x04". 
There is no comment in [central directory](https://en.wikipedia.org/wiki/ZIP_(file_format)#Central_directory_file_header) to detect.

Idk if it would make sense as it might conflict with a zip files in the future.

Reply:
It's okay here, don't worry. FileIO queries the file/io format in the following order:

1. check the extension `.npz`
2. check if the first few bytes match the magic bytes registered for `.npz` format.
3. if not, return `Format{:UNKNOWN}` or error.

Reply:
Ok, I added them.  
Either way I'm waiting on NPZ to approve this [MR](https://github.com/fhs/NPZ.jl/pull/50) before we can proceed. This is why I marked this as a draft.